example/src/examples/TiledBuilding.tsx

```diff
@@ -0,0 +1,158 @@
+  const [buildings, setBuildings] = useState<any>();
+  const [selectedBuilding, setSelectedBuilding] = useState<any>();
+  const [mapRegion, setMapRegion] = useState<any>();
+  
+  const [isLoading, setIsLoading] = useState<Boolean>(false); 
+  const [error, setError] = useState<String>();
+  const [floor, setFloor] = useState<any>();
+
+  const [hasStoredTiles, setHasStoredTiles] = useState<Boolean>(false); 
+  const [offlineTilePath, setOfflineTilePath] = useState<String>();
+  
+  const getBuildingInfo = (building: any) => {
+    
+    SitumPlugin.fetchBuildings(
+        (buildings) => {
+          // returns list of buildings
+          if (!buildings || buildings.length == 0)
+            alert(
+              'No buildings, add a few buildings first by going to:\nhttps://dashboard.situm.es/buildings',
+            );
+
+            for (const [key, b] of buildings.entries()) {
+                console.log(key + JSON.stringify(b));
+
+
+                if (b.buildingIdentifier === SITUM_BUILDING_ID) {
+                    console.log("Found selected buidling");
+                    console.log("Found required building, going to download entire building");
+                    SitumPlugin.fetchBuildingInfo(
+                        b,
+                        (buildingInfo: any) => {
+                          console.log('FetchBuildingInfo ' + JSON.stringify(buildingInfo));
+                
+                          if (buildingInfo.floors.length >= 1) {
+                            setFloor(buildingInfo.floors.find((fl: any) => fl.level === 0));
+                          } else {
+                            console.error('No floors found!');
+                          }
+                
+                
+                        setMapRegion( {
+                            latitude: buildingInfo.building.center.latitude,
+                            longitude: buildingInfo.building.center.longitude,
+                            latitudeDelta: 0.005,
+                            longitudeDelta: 0.005,
+                          });
+                
+                          getOfflineTiles(b);
+                        },
+                        (error: any) => {
+                          console.log('FetchBuildingInfoError ' + error);
+                        },
+                      );
+                } 
+
+
+            }
+
+
+        },
+        (error) => {
+          // returns an error string
+          console.log("Error: " + error);
+        },
+      );
+    
+    
+    
+    
+    
+    
+  }
+
+  const getOfflineTiles = (building: any) => {
+    SitumPlugin.fetchTilesFromBuilding(building,
+        (result: any) => {
+          console.log("result is" + JSON.stringify(result));
+
+            setOfflineTilePath(result.results);
+
+        },
+        (error: any) => {
+          console.log("Fetch tiles from building error" + error);
+        }
+      );
+  }
+ 
+  const initializeBuilding = () => {
```

Comment:
Este `initializeBuilding()` chámase sólo dende un sitio. Quizais poida eliminarse para facelo máis sencillo de ler?